HyperPlay-Gaming/hyperplay-desktop-client
# [FIX] GPTK Filtering and download

Small fixes to GPTK handling like:
- Fix filtering wine versions when downloading default wine
- Fix wine version checking loop on github dealing to API request block
- Chore: Remove Wine-Staging from listing since it does not adds much to current listing.


### How to test

- Start a fresh install of HP on macOS (or delete the `tools` folder only.
- After a couple of minutes check if GPTK is installed.
- Check the logs if it has set GPTK as the default
- Check Wine Manager to see if it will only show Wine-Crossover and GPTK now. Wine Staging was removed.

---

Use the following Checklist if you have changed something on the Backend or Frontend:
